Translations/font_tables.py

```diff
@@ -772,7 +900,7 @@ def get_small_font_map_cyrillic() -> Dict[str, bytes]:
         "А": b"\x7e\x09\x09\x09\x7e\x00",
         "Б": b"\x7f\x49\x49\x49\x31\x00",
         "В": b"\x7f\x49\x49\x49\x36\x00",
-        "Г": b"\x7f\x01\x01\x01\x01\x00",
+        "Г": b"\x7f\x49\x49\x49\x36\x00",
```

Comment:
Is this change accidental?

Reply:
Yes I accidentally cmd+v'd instead of cmd+c'ing and it got changed into a B